lib/rules/indent.js

```diff
@@ -741,6 +750,24 @@ module.exports = {
                 checkIndentInArrayOrObjectBlock(node);
             },
 
+            MemberExpression: function(node) {
+                if (isSingleLineNode(node)) {
+                    return;
+                }
+
+                var propertyIndent = getNodeIndent(node) + indentSize * options.MemberExpression;
+
+                var checkNodes = [node.property];
+
+                var dot = context.getTokenBefore(node.property);
+
+                if (dot.type === "Punctuator" && dot.value === ".") {
```

Comment:
Everywhere else, the checking for whether the node is the first item on the line is delegated to the `check*` functions, so I was following that pattern. They all just send everything to `check*()` and let that function figure out if it's the first thing on the line and needs to be checked or not.

Your suggested logic would work just fine too. If that would be less confusing, I can do that instead.